daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -417,7 +423,8 @@ func (l *logStream) Log(msg *logger.Message) error {
 		case l.messages <- msg:
 			return nil
 		default:
-			return errors.New("awslogs buffer is full")
+			atomic.AddInt32(&l.discardedMessages, 1)
```

Comment:
would it be worth adding an explanation as to why logs are being discarded ? (ie `buffer full` or some other issue)

Reply:
Sure, I will mention that the reason is because the buffer is full in the part that logs the warning

Reply:
done